### ISSUE TITLE
CTRL + e not working

### ISSUE THREAD

**Issue by rajinder-yadav** [Microsoft/vscode#63272]
Issue Type: <b>Bug</b>

Can't switch over to file explorer using [CTRL+E].

What happen is the 'e' is typing into the editor and try to hit ctrl+z or [esc] don't work and just create more garbage typing.

VS Code version: Code - Insiders 1.30.0-insider (df8ea54c5397e67b3e47df15a355bd04345c6934, 2018-11-15T06:14:28.366Z)
OS version: Linux x64 4.15.0-39-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Phenom(tm) II X4 940 Processor (4 x 800)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: unavailable_software<br>video_encode: unavailable_software<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 1|
|Memory (System)|15.66GB (3.59GB free)|
|Process Argv|/home/yadav/dev/docs/guides|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (22)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|9.2.0
dep|car|0.1.0
npm-intellisense|chr|1.3.0
vscode-markdownlint|Dav|0.21.0
vscode-eslint|dba|1.7.0
EditorConfig|Edi|0.12.5
tslint|eg2|1.0.40
markdown-toc|jof|1.4.0
solidity|Jua|0.0.46
sort-typescript-imports|mic|1.4.1
Angular-BeastCode|Mik|7.0.15
cpptools|ms-|0.20.1
debugger-for-chrome|msj|4.11.1
angular2-inline|nat|0.0.17
java|red|0.33.0
language-stylus|sys|1.10.0
sort-lines|Tyr|1.7.0
vscode-java-debug|vsc|0.15.0
vscode-java-pack|vsc|0.4.0
vscode-java-test|vsc|0.10.0
vscode-maven|vsc|0.11.1
better-align|wwm|1.1.6

(2 theme extensions excluded)

</details>

**Comment by alexr00** [comment]
Does this still happen if you disable all your extensions?

**Comment by David-Else** [comment]
You mean ctrl+shift + e? That is the Linux shortcut to switch to file explorer.

If that is what you meant, then it is a duplicate of https://github.com/Microsoft/vscode/issues/48480

There is a workaround there, or you need to update your distro. Fedora 29 fixed this issue for me.

**Comment by rajinder-yadav** [comment]
Yes I meant ctrl + shift + e and this happens also with plugins disabled. Below is what I see.

As per that other workaround, I am using KDE desktop which uses Qt5 not GTK.

![screenshot_20181116_173534](https://user-images.githubusercontent.com/439226/48650871-2a766a00-e9c6-11e8-805e-38e7118b6851.png)

**Comment by alexandrudima** [comment]
What does VS Code detect when you press the desired key combination? Please hover over the input box and take a screenshot:
![keyboard-input](https://cloud.githubusercontent.com/assets/5047891/25478333/fe31c830-2b40-11e7-865c-b2c3bb10dd75.gif)

**Comment by rajinder-yadav** [comment]
When I hover it says, "Type to search in keybinding"

![screenshot_20181117_214503](https://user-images.githubusercontent.com/439226/48668048-2293f400-eab2-11e8-8245-ba7c82d0f6ce.png)

**Comment by rajinder-yadav** [comment]
![out](https://user-images.githubusercontent.com/439226/48668153-cb435300-eab4-11e8-8c3e-21b5dcf11d9e.gif)

**Comment by rajinder-yadav** [comment]
That key combo works fine on my system.

![image](https://user-images.githubusercontent.com/439226/48812335-e5bd3c80-ecff-11e8-8718-6ead9c0a80de.png)

**Comment by rajinder-yadav** [comment]
Also If I type, "CTRL+SHIFT+G" or "CTRL+SHIFT+D" then "CTRL+SHIFT+E" the explorer does display.

However when I click in the terminal or editor, CTRL+SHIFT+E doesn't work. So this does seem to be buggy behaviour.

**Comment by rajinder-yadav** [comment]
Here is the keycode using `xev`

```
KeyPress event, serial 40, synthetic NO, window 0x5e00001,
    root 0x110, subw 0x0, time 76251782, (1139,543), root:(1139,572),
    state 0x0, keycode 105 (keysym 0xffe4, Control_R), same_screen YES,
    XLookupString gives 0 bytes: 
    XmbLookupString gives 0 bytes: 
    XFilterEvent returns: False

KeyPress event, serial 40, synthetic NO, window 0x5e00001,
    root 0x110, subw 0x0, time 76252991, (1139,543), root:(1139,572),
    state 0x4, keycode 62 (keysym 0xffe2, Shift_R), same_screen YES,
    XLookupString gives 0 bytes: 
    XmbLookupString gives 0 bytes: 
    XFilterEvent returns: False

KeyPress event, serial 40, synthetic NO, window 0x5e00001,
    root 0x110, subw 0x0, time 76255332, (1139,543), root:(1139,572),
    state 0x5, keycode 26 (keysym 0x45, E), same_screen YES,
    XLookupString gives 1 bytes: (05) ""
    XmbLookupString gives 1 bytes: (05) ""
    XFilterEvent returns: False

KeyRelease event, serial 40, synthetic NO, window 0x5e00001,
    root 0x110, subw 0x0, time 76255426, (1139,543), root:(1139,572),
    state 0x5, keycode 26 (keysym 0x45, E), same_screen YES,
    XLookupString gives 1 bytes: (05) ""
    XFilterEvent returns: False

KeyRelease event, serial 40, synthetic NO, window 0x5e00001,
    root 0x110, subw 0x0, time 76264323, (1139,543), root:(1139,572),
    state 0x5, keycode 62 (keysym 0xffe2, Shift_R), same_screen YES,
    XLookupString gives 0 bytes: 
    XFilterEvent returns: False

KeyRelease event, serial 40, synthetic NO, window 0x5e00001,
    root 0x110, subw 0x0, time 76264327, (1139,543), root:(1139,572),
    state 0x4, keycode 105 (keysym 0xffe4, Control_R), same_screen YES,
    XLookupString gives 0 bytes: 
    XFilterEvent returns: False
```

**Comment by rajinder-yadav** [comment]
This doesn't seem to be a good reliable test as the browser doesn't even work for cmd+e when I try this in chrome. I've tried different keys and chrome shortcut get executed in place :)

**Comment by alexandrudima** [comment]
On my machine, this is a reliable test, because ctrl+shift+e is not reserved on Chrome for Windows:

![image](https://user-images.githubusercontent.com/5047891/48943114-6d829100-ef22-11e8-834f-a3badc9b0931.png)

**Comment by alexandrudima** [comment]
@rajinder-yadav I believe you have an environmental problem. I can press ctrl+e just fine on Kubuntu. I am using Hyper-V.

![ctrl-e](https://user-images.githubusercontent.com/5047891/49141547-22d49080-f2f7-11e8-8b7e-795f9064c12d.gif)

**Comment by rajinder-yadav** [comment]
The bug exist for me in chrome, nothing happen in the playground got SHIFT+CTRL+E, but if I change the code to CTRL+E the pop is displayed.

**Comment by rajinder-yadav** [comment]
The problem must be with VSCode, it's doing something strange.

1. I showed using `xev` all the correct key events are generated.
2. I showed setting shortcut for other GUI apps, which are easily set to CTRL+SHIFT+E

Also why SHIFT+CTRL+E work after I press SHIFT+CTRL+D?

**Comment by alexandrudima** [comment]
@rajinder-yadav If you can reproduce the bug in Chrome, then this is an issue within Chromium, since Chrome contains no source code authored by us. Please go to https://bugs.chromium.org/p/chromium/issues/list

**Comment by rajinder-yadav** [comment]
Thanks @alexandrudima, can't be bother to create an account to log the bug. I'll live with it for now.

**Comment by smhnaji** [comment]
I can confirm that I see the problem in both of the following environments:

* On my laptop running Ubuntu 17.10 (tested with VSCode extensions, and after disabling extensions)
* On a very fresh Linux Mint 19 PC. Visual Studio Code Version: `1.30.0` (I have not yet installed Chrome or Chromium on it)

As accessing to File Explorer is vital for me, I went to *Keyboard Shortcuts* and removed `Ctrl+Shift+X` (you know, it's not so necessary). Then I changed the shortcut of `Show Explorer` and `workbench.action.quickOpenNavigatePreviousInFilePicker` from **Ctrl+Shift+E** to **Ctrl+Shift+X**.

It's now working with **Ctrl+Shift+X**.
